.github/ISSUE_TEMPLATE/documentation_issue.md

```diff
@@ -0,0 +1,19 @@
+---
+name: "Documentation Issue"
+about: "Report issues related to documentation"
+title: "[Docs] "
+labels: ["documentation"]
+assignees: []
+---
+
+### Description  
+<!-- What is unclear, incorrect, or missing? -->
+
+### Location of the Issue  
+<!-- Provide links or file paths where the issue exists. -->
```

Comment:
uncomment all of these (i'm not making a comment per line, but that is what I'm saying)